src/sections/shared/layout/app-loader/AppLoader.tsx

```diff
@@ -1,9 +1,17 @@
+import cn from 'classnames'
```

Comment:
Do we need a test for this class, for the appearance of the spinner?

Reply:
Yes you are right, I will add a component test and validate that the dynamic classes based on the prop actually change the css of the container.
Also I need to solve some merging conflicts.